presto-docs/src/main/sphinx/release/release-0.235.rst

```diff
@@ -0,0 +1,64 @@
+=============
+Release 0.235
+=============
+
+General Changes
+_______________
+* Fix named queries where its output does not match its column list (:issue:`14333`).
+* Fix error in analysis phase.
+* Fix ZSTD compression issue with zero row file for missing buckets.
```

Comment:
Did it cause query failure?

Reply:
Yes

Reply:
Yes. Queries were failing.

Reply:
@tdcmeehan Shouldn't this be hive connector change?